test/unit/util.js

```diff
@@ -1058,11 +1058,11 @@
     });
   };
 
-  QUnit.test('decodeTransformMatrix with angle', function (assert) {
-    for (let angle = 0; angle <= 360; angle += 30) {
+  for (let angle = 0; angle <= 360; angle += 30) {
+    QUnit.test(`decodeTransformMatrix with ${angle}`, function (assert) {
```

Comment:
This is SO weird!

Reply:
How did you even understand that this is what makes it work????

Reply:
Did you try renaming assert.close? Might it be a name collision?

Reply:
> Did you try renaming assert.close? Might it be a name collision?

I tried too, it didn't work

> How did you even understand that this is what makes it work????

It seems to me that too many asserts in a single test cause this problem in firefox/chrome. A simple test like this gives the same problem.

```
  QUnit.test.only('many assertions in a single test', function (assert) {
    for (let i = 0; i <= 28000; i += 1) {
      assert.equal(i, i, 'msg')
    }
  });
```